docker-build-context/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Goreleaser builds a docker image according to this Dockerfile by copying the sysl binary 
+# to a golang:1.13-alpine image and setting up the entrypoint.
+#
+# The produced image is published to https://hub.docker.com/r/anzbank/sysl
+ARG go_ver=1.14
```

Comment:
The current `go.mod` has 1.13. Bumping to 1.14 should be fine, but should not be part of this PR.